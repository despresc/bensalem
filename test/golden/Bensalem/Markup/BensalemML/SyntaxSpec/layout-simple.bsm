@&layout
  with some content
     and more
   here
