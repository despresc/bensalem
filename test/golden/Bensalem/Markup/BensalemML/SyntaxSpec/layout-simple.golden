fromList
 [ ElementNode
  ( Element
   { elementTagPos = SrcSpan
    { srcSpanName = ""
    , srcSpanStart = SrcPos
     { srcOffset = 0
     , srcLine = 1
     , srcCol = 1
     }
    , srcSpanEnd = SrcPos
     { srcOffset = 8
     , srcLine = 1
     , srcCol = 9
     }
    }
   , elementArgPresentation = PresentLayout
   , elementName = "layout"
   , elementAttrs = fromList []
   , elementArg = fromList
    [ LineEnd
    , LineSpace 2
    , PlainText "with"
    , LineSpace 1
    , PlainText "some"
    , LineSpace 1
    , PlainText "content"
    , LineEnd
    , LineSpace 5
    , PlainText "and"
    , LineSpace 1
    , PlainText "more"
    , LineEnd
    , LineSpace 3
    , PlainText "here"
    ]
   }
  )
 , LineEnd
 ]